### PR TITLE
[mlir]: clarify tensor.pad docs for low/high config

### PR DESCRIPTION
**Pull request by j2kun**
Thanks @makslevental for educating me ^_^

### CODE REVIEW

**Review by makslevental — APPROVED** [review]
Man talk about annoying - I can't add this comment because it's not a changed line; can you change 

```
The result tensor dimensions are `low` + `dim` + `high` along that
    dimension.
```

on line 1177 to say

```
The result tensor dimensions are `low[i]` + `dim[i]` + `high[i]` for each dimension `i`.
```

**Comment by j2kun** [comment]
> Man talk about annoying - I can't add this comment because it's not a changed line; can you change
> 
> ```
> The result tensor dimensions are `low` + `dim` + `high` along that
>     dimension.
> ```
> 
> on line 1177 to say
> 
> ```
> The result tensor dimensions are `low[i]` + `dim[i]` + `high[i]` for each dimension `i`.
> ```

Fixed!

File: mlir/include/mlir/Dialect/Tensor/IR/TensorOps.td [hunk 1]
```diff
@@ -1219,10 +1234,11 @@ def Tensor_PadOp : Tensor_Op<"pad", [
       } : tensor<2x3xf32> to tensor<?x?xf32>
     ```
 
-    Example 4:
+    Example 5: Force a padded value to be always exist with `nofold`, even
+    though the padding config asserts no new elements will be added to the
```
Comment (ftynse) [inline mlir/include/mlir/Dialect/Tensor/IR/TensorOps.td:1238]: Nit: `assert` sounds like it will trigger an assertion failure :)

**Review by ftynse — APPROVED** [review]

File: mlir/include/mlir/Dialect/Tensor/IR/TensorOps.td (hunk 1 above)
Comment (j2kun) [reply mlir/include/mlir/Dialect/Tensor/IR/TensorOps.td]: s/asserts/specifies/

**Comment by j2kun** [comment]
IIUC, with a single-commit, a rebase+merge will preserve authorship while a squash+merge will overwrite the commit email. See https://discourse.llvm.org/t/hidden-emails-on-github-should-we-do-something-about-it/74223

**Comment by ftynse** [comment]
Rebase+merge is not allowed in this repo...

**Review by MaheshRavishankar — APPROVED** [review]
Thanks for the doc update!

**Comment by j2kun** [comment]
OK to squash+merge then with anonymous email?

**Comment by ftynse** [comment]
Landed manually as 6a2f68beb7c9cbe8ab1299c9feac32c42affdb03.

**Pull request closed without merging**